Dockerfiles/Dockerfile.documentation

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.10.3
+
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Replace with: `go get -u github.com/golang/dep/cmd/dep`

Reply:
all other docker files install dep in this manner.  Should I refactor all of them?

Reply:
Yeah, if its not too much of a lift.  The ops files use the `go get` approach.